packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/di/app_dependencies.dart

```diff
@@ -87,13 +86,20 @@ class AppDependencies {
       ];
 
   List<Provider> get _httpClients => [
-        Provider<Dio>(
+        Provider<PlainHttpClient>(
+          create: (context) {
+            final client = PlainHttpClient();
+            // Configure baseUrl if needed.
+            return client;
+          },
+        ),
+        Provider<ApiHttpClient>(
           create: (context) {
-            final dio = Dio()
+            final client = ApiHttpClient()
               ..options.baseUrl = Platform.isIOS
-                  ? config.iosSimulatorBaseApiUrl
-                  : config.androidEmulatorBaseApiUrl;
-            return dio;
+                ? config.iosSimulatorBaseApiUrl
```

Comment:
I believe this is very specific to the counter example so it would be great if it's encapsulated within the appConfig.

Reply:
Implemented.